src/Validation/Validation.php

```diff
@@ -1021,6 +1023,74 @@ public static function uploadedFile($file, array $options = [])
     }
 
     /**
+     * Validates a geographic coordinate.
+     *
+     * Supported formats are right now:
+     *
+     * - <latitude>, <longitude> Example: -25.274398, 133.775136
+     *
+     * ### Options
+     *
+     * - `type` - A string of the coordinate format, right now only `latLong`.
```

Comment:
Go to [this page](http://www.earthpoint.us/Convert.aspx) and scroll down. Below the calculator there are different formats explained.